flytekit/extend/backend/base_agent.py

```diff
@@ -149,6 +150,10 @@ def is_terminal_state(state: State) -> bool:
     return state in [SUCCEEDED, RETRYABLE_FAILURE, PERMANENT_FAILURE]
 
 
+def get_secret(secret_key: str) -> str:
```

Comment:
nit: could we rename it to `get_agent_secret`

Reply:
no problem

Reply:
Done